website/src/components/CommunitySlide/CommunitySlide.css

```diff
@@ -0,0 +1,46 @@
+.communityCard {
+    width: 390px;
```

Comment:
prefer `rem` over `px` in css. Border radius values with `px` are okay.

Reply:
Done ✅